chrome/app/scripts/contentscript.js

```diff
@@ -69,31 +69,30 @@ var buildContents = function(links) {
   return contents;
 };
 
-var insertContents = function(contents) {
+var insertContents = function (contents) {
```

Comment:
After this pull request I'll setup Prettier so this auto-formats consistently going forward.

Reply:
ok，My code is idea format .
It's different from your way.